src/client/theme-default/components/VPSidebar.vue

```diff
@@ -25,6 +25,11 @@ watch(
   },
   { immediate: true, flush: 'post' }
 )
+
+// scroll active link into view
+onMounted(() => {
+  nextTick(() => navEl.value?.querySelector('.is-active')?.scrollIntoView({ block: 'center'}))
```

Comment:
And if the same route or pattern is placed 2+ times in the sidebar, is it working fine? also I'd calculate to scroll only when it is out of the viewport